web/style/home.scss

```diff
@@ -0,0 +1,233 @@
+@import '_variables.scss';
+@import 'common.scss';
+
+@mixin border-bottom-radial {
+  position: relative;
+  padding-bottom: 39px;
+  &:after {
+    content: '';
+    bottom: -1rem;
+    background: radial-gradient(farthest-corner at 50% 0px, #d4d2d2 0%, transparent 50%);
+    width: 100%;
+    height: 1rem;
+    position: absolute;
+    bottom: 0;
+  }
+}
+
+main.home {
+  .offer-banner {
+    @include border-bottom-radial;
+
+    &__img {
+      display: flex;
+      flex: 1;
+      width: 100%;
+    }
+  }
+  .category {
+    @include border-bottom-radial;
+
+    &:nth-child(odd) {
+      flex-direction: row-reverse;
+    }
+    .category-logo {
+      height: 200px;
+      max-width: 50%;
+    }
+    .category-detail {
+      align-items: center;
+      justify-content: center;
+      text-align: center;
+      &__main {
+        font-size: 1.8rem;
+        margin-bottom: 10px;
+      }
+      &__sub {
+        font-size: 0.9rem;
+        margin-bottom: 10px;
+      }
+    }
+  }
+}
+
+
+.slideshow-container img {
+  margin-left: auto;
+  margin-right: auto;
+  display: block;
+}
+.mySlides {display: none}
+img {vertical-align: middle;}
+
+/* Slideshow container */
+.slideshow-container {
+ max-width: 1000px;
+ position: relative;
+ margin: auto;
+}
+
+/* Next & previous buttons */
+.prev, .next {

```

Comment:
This CSS is for carousel? What if you need carousel on cart page too?

Reply:
Separated done